api/v1/observer/observer.proto

```diff
@@ -174,3 +230,25 @@ message TLS {
     // cert validation process.
     string server_name = 2;
 }
+
+// ExportEvent contains an event to be exported.
```

Comment:
Can we add a warning comment that this protobuf message is not intended to be used outside of the export feature please?